carrot_cli/src/carrot_cli/rest/runs.py

```diff
@@ -52,7 +63,16 @@ def find(
         ("sort", sort),
         ("limit", limit),
         ("offset", offset),
+        ("csv", str(csv is not None).lower())
```

Comment:
It's actually checking if `csv` has a value, converting the boolean result of that check to a string, and making it lowercase because python booleans are True/False, but carrot expects true/false